tests/vlan/test_vlan.py

```diff
@@ -118,6 +118,19 @@ def setup_vlan(ptfadapter, duthosts, rand_one_dut_hostname, ptfhost, vlan_ports_
             duthost.command('config vlan add {}'.format(vlan['vlan_id']))
             duthost.command("config interface ip add Vlan{} {}".format(vlan['vlan_id'], vlan['ip'].upper()))
 
+        # Delete untagged vlans from interfaces to avoid error message
+        # when adding untagged vlan to interface that already have one
+        if 'master' in duthost.os_version or '202012' in duthost.os_version:
```

Comment:
Is there any harm in doing this on all OS versions? It's definitely needed for 202012 and master, but do we need to preserve the logic for older versions?

Reply:
@daall Yes, on 201911 images this change will break the test, as for show vlan config looks like this:
```
show vlan config                                                                                                                                    
Name        VID                                                                                                                                                          
--------  -----                                                                                                                                                          
Vlan1000   1000
```
And eventually we will received `Error: Ethernet8 is not a member of Vlan1000`

Reply:
Got it! In that case it might be more future-proof to check "if 201911 not in duthost.os_version" for this part. Thoughts?

Reply:
Good idea, done

Reply:
@AndriiLozovyi Is this a 201911 image bug that "on 201911 images this change will break the test"?
(+@ganglyu for awareness)